src/add-to-registry.js

```diff
@@ -22,7 +22,6 @@ const { isInRegistry, addToRegistry, getFromRegistry, updateInRegistry, TEMPLATE
             "description": npmPackageMetadata.description,
             "latestVersion": npmPackageMetadata.version,
             "publishDate": (new Date(Date.now())).toISOString(),
-            "services": npmPackageMetadata.services,
```

Comment:
We need to change it to "categories" as categories are mandatory in `install.yml`.
And an `if` block below
```
        if (npmPackageMetadata.categories) {
            templateData['categories'] = npmPackageMetadata.categories;
        }
```
change to
```
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```

We will have
```
        let templateData = {
            "author": npmPackageMetadata.author,
            "name": npmPackageMetadata.name,
            "description": npmPackageMetadata.description,
            "latestVersion": npmPackageMetadata.version,
            "publishDate": (new Date(Date.now())).toISOString(),
            "categories": npmPackageMetadata.categories,
            "adobeRecommended": adobeRecommended,
            "keywords": npmPackageMetadata.keywords,
            "status": TEMPLATE_STATUS_APPROVED,
            "links": {
                "npm": npmUrl,
                "github": gitHubUrl
            }
        }
        if (npmPackageMetadata.extension) {
            templateData['extension'] = npmPackageMetadata.extension;
        }
        if (npmPackageMetadata.apis) {
            templateData['apis'] = npmPackageMetadata.apis;
        }
```